src/api/db/client.ts

```
@@ -1,8 +1,13 @@
-  connection: {
-    url: `${process.env.TURSO_DATABASE_URL}`,
-    authToken: `${process.env.TURSO_AUTH_TOKEN}`,
-  },
-})
+const connection =
+  process.env.NODE_ENV === 'test'
+    ? {
+        url: 'file:test.db',
+      }
+    : {
+        url: process.env.TURSO_DATABASE_URL!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)